pkg/review/commit_review_status_test.go

```diff
@@ -0,0 +1,443 @@
+                   "totalCount": 2
+                 }
+               }
+             }
+           }
+         }`,
+			},
+		},
+		{
+			name:       "No Associated Pull Requests for a commit",
+			token:      "ghp_JQqAJky0GlNB7xqVCbUrivIgFZ09V8gGCi5C",
+			githubOrg:  "test-org",
+			repository: "test-repo",
+			commitSha:  "kof6p96lr6qvdu81qw49fhmoxrod9qmc2qak51nh",
+			wantRequestBodies: []string{
+				`{
+           "query": "
+             query($commitSha:GitObjectID! $githubOrg:String! $pageCursor:String! $repository:String!) {
+               repository(owner: $githubOrg, name: $repository) {
+                 object(oid: $commitSha) {
+                   ... on Commit{
+                     associatedPullRequests(first: 100, after: $pageCursor) {
+                       nodes{
+                         databaseId,
+                         number,
+                         reviewDecision
+                       },
+                       pageInfo{
+                         endCursor,
+                         hasNextPage,
+                         hasPreviousPage,
+                         startCursor
+                       },
+                       totalCount
+                     }
+                   }
+                 }
+               }
+             }
+           ",
+           "variables": {
+             "commitSha": "kof6p96lr6qvdu81qw49fhmoxrod9qmc2qak51nh",
+             "githubOrg": "test-org",
+             "pageCursor": "",
+             "repository":"test-repo"
+           }
+         }`,
+			},
+			want: []*PullRequest{},
+			responseBodies: []string{
+				`{
+           "data": {
+             "repository": {
+               "object": {
+                 "associatedPullRequests": {
+                   "nodes": [],
+                   "pageInfo": {
+                     "endCursor": "XQ",
+                     "hasNextPage": false,
+                     "hasPreviousPage": false,
+                     "startCursor": ""
+                   },
+                   "totalCount": 0
+                 }
+               }
+             }
+           }
+         }`,
+			},
+		},
+	}
+	for _, tc := range cases {
+		requestNumber := 0
+		fakeGitHub := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+			expectedAuthHeader := fmt.Sprintf("Bearer %s", tc.token)
+			if r.Header.Get("Authorization") != expectedAuthHeader {
+				w.WriteHeader(500)
+				fmt.Fprintf(w, "missing or malformed authorization header")
+				return
+			}
+			bytes, _ := io.ReadAll(r.Body)
+			body := string(bytes)
+			wantBody := tc.wantRequestBodies[requestNumber]
+			if !equalExceptWhiteSpace(body, wantBody) {
+				t.Errorf("Incorrect Request Body: got=%s, want=%s", body, wantBody)
+			}
+			fmt.Fprintf(w, tc.responseBodies[requestNumber])
+			requestNumber++
+		}))
+		src := oauth2.StaticTokenSource(
+			&oauth2.Token{AccessToken: tc.token},
+		)
+		httpClient := oauth2.NewClient(context.Background(), src)
```

Comment:
nit: create a shared `context.Background()` you don't have to repeat it here and below

Reply:
fixed.